examples/tv-app/tv-common/clusters/messages/MessagesManager.h

```diff
@@ -44,5 +129,5 @@ class MessagesManager : public chip::app::Clusters::Messages::Delegate
     uint32_t GetFeatureMap(chip::EndpointId endpoint) override;
 
 protected:
-    std::list<chip::app::Clusters::Messages::Structs::MessageStruct::Type> mMessages;
+    std::list<CachedMessage> mCachedMessages;
```

Comment:
Does this use copy-constructors? if so could this double-free?

Reply:
Checked to confirm no double free in my testing.